io/pson/pson.go

```diff
@@ -0,0 +1,46 @@
+package pson
```

Comment:
Is PSON an alternative to JSON or something? P isn't really around J so I'm wondering if this is not a typo.

Reply:
> poly json thought I wrote a comment about it but essentially the namespace collisions with Go's json package was too much of a pain. Should make a comment if there isn't already

Reply:
See -> https://github.com/TimothyStiles/poly/pull/174#discussion_r661936455